packages/forms-web-app/src/validators/shared/telephone-number.js

```diff
@@ -0,0 +1,26 @@
+const { body } = require('express-validator');
+
+const rules = () => {
```

Comment:
Wondering if this can just be an array instead of a function although I might be missing something